src/Adyen/Service/ResourceModel/Checkout/Refunds.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Adyen\Service\ResourceModel\Checkout;
+
+class Refunds extends \Adyen\Service\AbstractCheckoutResource
+{
+    /**
+     * @var string
+     */
+    protected $endpoint;
+
+    /**
+     * Include applicationInfo key in the request parameters
+     *
+     * @var bool
+     */
+    protected $allowApplicationInfo = false;
+
+    /**
+     * Payments constructor.
+     *
+     * @param \Adyen\Service $service
+     * @throws \Adyen\AdyenException
+     */
+    public function __construct($service)
+    {
+        $this->endpoint = $this->getCheckoutEndpoint($service) .
+            '/' . $service->getClient()->getApiCheckoutVersion() . '/payments/{paymentPspReference}/refunds';
```

Comment:
How do you pass the paymentPspReference in the url ?

Reply:
Hi @AlexandrosMor, I pass it as a request param, see: `\Adyen\Service\AbstractResource::replacePathParameters`

If there is a match then I replace it.

You can check the example in the integration test:
`\Adyen\Tests\Integration\CheckoutTest::testRefunds`